dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -122,14 +138,14 @@ def authenticate
       nrps_url = decoded_jwt[Policies::Lti::LTI_NRPS_CLAIM][:context_memberships_url]
       resource_link_id = decoded_jwt[Policies::Lti::LTI_RESOURCE_LINK_CLAIM][:id]
       deployment_id = decoded_jwt[Policies::Lti::LTI_DEPLOYMENT_ID_CLAIM]
-      deployment = Queries::Lti.get_deployment(integration.id, deployment_id)
+      deployment = Queries::Lti.get_deployment(integration[:id], deployment_id)
```

Comment:
Since we are now getting the `integration` back from either the cache or the db, I had to change to bracket notation, since the cache doesn't return a class object like Active Record does, and therefore doesn't support dot notation.

Reply:
Does the bracket + symbol notation work for both cached and ActiveRecord values?

Reply:
It does, ActiveRecord has a `[]` method (see [here](https://api.rubyonrails.org/classes/ActiveRecord/AttributeMethods.html#method-i-5B-5D)). I don't see a way around this if we implement the cache.